jetty-websocket/websocket-jetty-common/src/main/java/org/eclipse/jetty/websocket/common/JettyWebSocketFrameHandler.java

```diff
@@ -283,12 +283,19 @@ private void onCloseFrame(Frame frame, Callback callback)
     @Override
     public void onClosed(CloseStatus closeStatus, Callback callback)
     {
+        Callback suspendedCallback;
         try (AutoLock l = lock.lock())
         {
             // We are now closed and cannot suspend or resume.
             state = SuspendState.CLOSED;
+            delayedFrame = null;
+            suspendedCallback = delayedCallback;
```

Comment:
Not a big fan of mixing the `suspended` and `delayed` naming here.   How about calling the local variable `theDelayedCallback`.  (Note that is why I like the underscore prefix!)